test/tests/functional/pbs_acl_host_moms.py

```diff
@@ -52,40 +52,40 @@ def setUp(self):
 
         TestFunctional.setUp(self)
 
-        if len(self.moms) != 2:
-            self.skip_test('test requires two MoMs as input, ' +
-                           '  use -p moms=<mom1>:<mom2>')
+        if len(self.moms) != 1:
+            self.skip_test('test requires one MoM as input, ' +
+                           '  use -p moms=<mom>')
```

Comment:
Yes, have added the logs to the ticket. Had to modify it a little to skip when client was not specified.

Also, I've been made aware that removing hooks should be part of the PTL framework and need not be handled explicitly by the user, so I'm reverting the code back to not deleting the hook after the test case.